valhalla/baldr/double_bucket_queue.h

```diff
@@ -82,16 +112,25 @@ class DoubleBucketQueue final {
   void clear() {
     // Empty the overflow bucket and each bucket
     overflowbucket_.clear();
-    while (currentbucket_ != buckets_.end()) {
```

Comment:
this seems to suggest that we dont need to clear them all

Reply:
Normal operation will end up pop'ing elements from all lower cost buckets so this means we need to only clear buckets >= the current cost bucket.

Reply:
Thank you, I will remove this change.